wheatley/row_generation/complib_composition_generator.py

```diff
@@ -127,7 +133,12 @@ def summary_string(self) -> str:
         """ Returns a short string summarising the RowGenerator. """
         return f"{'private ' if self.is_comp_private else ''}comp #{self.comp_id}: {self.comp_title}"
 
-    def _gen_row(self, previous_row: Row, stroke: Stroke, index: int) -> Row:
+    def _gen_row(self, _previous_row: Row, _stroke: Stroke, index: int) -> Row:
```

Comment:
Why have the parameter names changed? (I guess they're not used in this overload of the abstract method)

Reply:
The underscores are to tell pylint that it's OK that the variables aren't used.  Thinking about it, I've only seen this convention used in other languages like Haskell and Rust (both of which have `_` as a variable name that can be assigned to arbitrarily many times but never read from), so I'm not sure how pythonic it is.  But it seems to shut pylint up.